app/utils/index.ts

```diff
@@ -0,0 +1,18 @@
+class Generator {
+  private *idGenerator(start: number, end: number, step: number) {
+    for (let i = start; i < end; i += step) {
```

Comment:
The requirement states that both boundaries are inclusive. This is a good test case to have.